plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -118,6 +118,13 @@ contract RootChain {
         addExitToQueue(depositPos, msg.sender, amount);
     }
 
+    function startFeeExit(uint256 amount)
```

Comment:
This seemingly gives the authority the ability to exit any amount from the contract and always be placed at the front of the queue. Do we need a more robust fee exiting mechanism?